### ISSUE TITLE
lit-html 2

### ISSUE THREAD

**Issue by alfonsogarciacaro** [AngelMunoz/Fable.Haunted#6]
This is a bit confusing, I realized the documentation in lit.dev is for lit-html 2 which is currently in release candidate state. And haunted depends on lit-html 1. It'd be nice to start using lit-html 2 right away as it has interesting features like `ref` but I'm not sure when haunted will support it. Besides, I was very excited about virtual components because they would make the transition easier from react components, but it looks [they have issues](https://github.com/matthewp/haunted/issues/69#issuecomment-485167789).

So basically the question is, how much should we commit ourselves to haunted? Does it look as they will move fast enough? Should we try to create our own hooks (with the new directives in lit-html 2 this looks easier).

**Comment by AngelMunoz** [comment]
Work here has been done to ensure we use the lit-html@2
